views.go

```diff
@@ -567,6 +567,10 @@ func (lk *logKeeper) findLogs(ctx context.Context, query bson.M, sort bson.D, mi
 				}
 				lineNum++
 			}
+
+			if ctx.Err() != nil {
```

Comment:
Shouldn't this check the DB context error?

Reply:
I think this is the same discussion as the previous thread. Let's continue the discussion there and I'll change both if we decide they need to change.